### PR TITLE
GTEx Documentation

### PR DESCRIPTION
**Pull request by Ayush-Praveen**

### CODE REVIEW

File: docs/Data Lake.md [hunk 1]
```diff
@@ -283,6 +285,66 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
 Single Cell Visualization provides a visualization interface for single cell data hosted on GEO. The data lake working on its backend is updated regularly with new datasets. The app has different visualizations for understanding data: *Dimensionality reduction plot*, *Violin plot*, *Dot Plot*. 
 -->
 
+##GTEx 
+
+**Opening the app**
+
+Using GTEx card on the Discover portal GTEx tissues can be accessed.
```
Comment (avijitzutshi) [inline docs/Data Lake.md:292]: rephrase and remove portal. Just mention it as Discover.

File: docs/Data Lake.md [hunk 2]
```diff
@@ -283,6 +285,66 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+
+As the application starts, it will load the requested dataset. Once it is loaded, it can be explored. 
+
+*   **Principal Component Analysis (PCA)**
+
+Principal Component Analysis: Also known as PCA plot, it is used to see the overall differences between cohorts of interest, if a strong separation is found along X axis (PC1) then that means strong biological differences between cohorts of interest. It provides an aspect to check the quality control of different samples. 
+
+![Metadata Table](../img/Discover/GTEx_4.png)
+
+The tab provides a metadata table to check different characterstics of samples. Further various parameters of PCA can be adjusted.
```
Comment (avijitzutshi) [inline docs/Data Lake.md:314]: "Further ... " to "Furthermore ... "

File: docs/Data Lake.md [hunk 3]
```diff
@@ -283,6 +285,66 @@ Clicking on *Plot* processes the data and the result of the query is a heatmap w
...
+Upon selecting the *Pathway Visualization* option, pathway specific genes can be selected. At a given time more than one pathways can be selected  
+
+![Pathway Visualization](../img/Discover/GTEx_8.png)
+![Pathway Visualization Plot](../img/Discover/GTEx_9.png)
+    
+*   **GTEx Expression Map**
+
+*GTEx Expression Map* can be used to explore the distribution of selected genes in different GTEx tissues. After exploring the selected tissue and finding list of genes of interest, it presents visualization methods like *GTEx Expression Violin* and *GTEx Expression Heatmap* to study the distribution of genes across different tissues. 
+
+A Single gene can be selected to plot violin for it's expression across different tissues.
```
Comment (avijitzutshi) [inline docs/Data Lake.md:339]: lower case "S"

**Review by avijitzutshi — CHANGES_REQUESTED** [review]
added comments

**Comment by Ayush-Praveen** [comment]
Hello @avijitzutshi , I have addressed your comments, can you have a look. Thanks

**Review by avijitzutshi — APPROVED** [review]

**Pull request merged**